### PR TITLE
Add the docs on testing and debugging the Intellij plugin

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
This is to add the docs on testing and debugging the Intellij plugin.

- Any checklist items to be verified before production deployment?
- Special styling concerns? 
- Related media files?

### CODE REVIEW

File: website/two-column-pages/docs/learn/tools-ides/intellij-plugin/testing-and-debugging-the-plugin.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+![select menu](../images/select-menu.png)
+
+3. Browse and select the **<CLONED_BALLERINA_DIRECTORY>/tool-plugins/intellij/build.gradle** file and click **Open**.
+
+4. Select **Use default gradle wrapper (recommended)** and then click **OK**.
+
+![select Gradle home](../images/select-gradle-home.png)
+
+## Testing and debugging the project
```
Comment (NipunaRanasinghe) [inline website/two-column-pages/docs/learn/tools-ides/intellij-plugin/testing-and-debugging-the-plugin.md]:
```suggestion
## Testing and debugging the project
```
```suggestion
## Testing and debugging the plugin
```

File: website/two-column-pages/docs/learn/tools-ides/intellij-plugin/testing-and-debugging-the-plugin.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+1. In the IntelliJ top menu, click **View**, click **Tool Windows**, and then click **Gradle** to launch the **Gradle projects** window.
+
+2. In the **Gradle projects** window, navigate to **Tasks** **->** **intellij** **->** **RunIde**.
```
Comment (NipunaRanasinghe) [inline website/two-column-pages/docs/learn/tools-ides/intellij-plugin/testing-and-debugging-the-plugin.md]:
```suggestion
2. In the **Gradle projects** window, navigate to **Tasks** **->** **intellij** **->** **runIde**. 
```

File: website/two-column-pages/docs/learn/tools-ides/intellij-plugin/testing-and-debugging-the-plugin.md (hunk 1 above)
Comment (praneesha) [reply website/two-column-pages/docs/learn/tools-ides/intellij-plugin/testing-and-debugging-the-plugin.md]: Fixed.

File: website/two-column-pages/docs/learn/tools-ides/intellij-plugin/testing-and-debugging-the-plugin.md (hunk 2 above)
Comment (praneesha) [reply website/two-column-pages/docs/learn/tools-ides/intellij-plugin/testing-and-debugging-the-plugin.md]: Fixed.

**Pull request merged**